README.md

```diff
@@ -7,7 +7,19 @@ software licenses that are associated with each dependency in the project.
 The offline cache allows spandx to perform a truly airgap friendly scan of software
 projects.
 
-![badge](https://github.com/mokhan/spandx/workflows/ci/badge.svg)
+### Supported project types
+
+Spandx can work with following language's package managers. It utilises lock files generated by package managers to find dependencies.
+
+| Language  | Package Manager    | Tested in |
+| ------------ | --------------- | -------:|
+| Ruby         | bundler         | 1.17.3  |
+| Js           | Npm             | 6.13.4  |
+| Js           | Yarn            | 1.19.1  |
+| Python       | Pypi            | <>      |
+| C#           | nuget           | <>      |
```

Comment:
C# is tricky because we cover *.sln, *.csproj and packages.config which isn't tied to a specific version of .NET. I'm not sure what to fill in the blanks with.